src/org/thoughtcrime/securesms/loki/redesign/activities/LinkedDevicesActivity.kt

```diff
@@ -145,12 +145,12 @@ class LinkedDevicesActivity : PassphraseRequiredActionBarActivity, LoaderManager
 
     override fun onDeviceLinkRequestAuthorized(deviceLink: DeviceLink) {
         LokiFileServerAPI.shared.addDeviceLink(deviceLink).success {
-            signAndSendDeviceLinkMessage(this, deviceLink).success {
+            signAndSendDeviceLinkMessage(this, deviceLink).successUi {
```

Comment:
If you're going to go with this instead of `Util.runOnMain` then do it below as well